bundle/metadata/annotations.yaml

```diff
@@ -4,12 +4,8 @@ annotations:
   operators.operatorframework.io.bundle.manifests.v1: manifests/
   operators.operatorframework.io.bundle.metadata.v1: metadata/
   operators.operatorframework.io.bundle.package.v1: cluster-logging
-  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.5
+  operators.operatorframework.io.bundle.channels.v1: stable,stable-5.6
```

Comment:
channel is still "stable"? does this get channed with the overlay?

Reply:
Fixed - use the overlay name as the channel name.